build.sh

```diff
@@ -18,6 +18,14 @@ usage() {
   echo -e "\tV1210                  - build Dasharo for Protectli V1210"
   echo -e "\tV1410                  - build Dasharo for Protectli V1410"
   echo -e "\tV1610                  - build Dasharo for Protectli V1610"
+  echo -e "\tV560TU                 - build Dasharo for Noacustom V560TU"
```

Comment:
```suggestion
  echo -e "\tV560TU                 - build Dasharo for Novacustom V560TU"
```

and all other instances